dbt_sugar/core/task/base.py

```diff
@@ -49,7 +49,9 @@ def __init__(
         self.dbt_tests: Dict[str, List[Dict[str, Any]]] = {}
         self.build_descriptions_dictionary()
 
-    def get_connector(self) -> Union[PostgresConnector, SnowflakeConnector, RedshiftConnector]:
+    def get_connector(
```

Comment:
this project allows for a line length of 100 chars. I would rather leave the pre-commit hook take care of the formatting as it will standardise it for everyone. Could you please revert this change?